playbook/lib/generators/kit/templates/kit_html.erb.tt

```diff
@@ -1,3 +1,12 @@
+<%%= pb_content_tag(:div,
+    aria: object.aria,
+    class: object.classname,
+    data: object.data,
+    id: object.id,
+    **combined_html_options) do %>
+  <span><%= @kit_name_uppercase %> CONTENT</span>
+<%% end %>
+
```

Comment:
@markdoeswork I wasn't sure if this file should replicate [PLAY-1305's](https://github.com/powerhome/playbook/blob/aa2dc2baf9f0069e288539a80ce2127e929fa4b6/playbook/lib/generators/kit/templates/kit_html.erb.tt) implementation exactly replacing the original content_tag with pb_content_tag or if both should be present in the file we are since not replacing all instances of it in one fell swoop/one PR.

Reply:
we can do this one in a different story imo